recipe/meta.yaml

```diff
@@ -1,59 +1,53 @@
 {% set name = "prophet" %}
-{% set version = "1.1.4" %}
+{% set version = "1.1.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 7f91e4ba322786e9512158783800b6eafb002200a7b3e9eadc13d7ff4d6e0634
-  patches:
-    - setup-make-folder.patch
+  sha256: 80973c0b8a22d835bfa9d6665a78ebc63115135eaef0f73b46ee14e9bad3ca1a
 
 build:
   number: 0
   # LunarCalendar, which is a noarch package, requires ephem, unavailable on s390x
   # https://blog.quantinsti.com/installing-prophet-library-windows/
   skip: true  # [py<37 or (linux and s390x)]
+  # because of missing numpy
+  skip: true  # [py>311]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   binary_has_prefix_files:
     - {{ SP_DIR|replace("\\", "/") }}/stan_model/prophet_model.bin
   # To use our own cmdstan rather than download and build a different one:
   script_env:
     - PROPHET_REPACKAGE_CMDSTAN=false
     - BOOST="${LIBRARY_INC}"  # [win]
+  missing_dso_whitelist:
+    - lib/libtbb.12.dylib   # [osx]
+    - lib/libtbb.so.12      # [linux]
```

Comment:
My comment still stands. Why do we need to add `lib/libtbb.so.12`?

Reply:
@boldorider4 tbb-devel should be put back in host. As a rule of thumb, it is better to over depend (i.e. include stuff you don't need) than over link (fail to include stuff that you actually need). If an over-linking check fails, the two options are to add the required dependency to the run section, or to ignore the check by adding to the missing_dso_whitelist. The cases where we would want to ignore the check are libraries like the linker so on s390x, but typically we should correct the run section.
As to why we get this here. cmdstan does depend on tbb and tbb-devel, so they both are in the host environment, but their run_export is not processed. Apparently we do have to explicitly add tbb-devel to host to get its run_export processed.

Reply:
Understood. Very insightful. Yes, I do see the need for `tbb-devel` in `host`. I always thought that because the runtime dependency on `libtbb.so.12` is only found at runtime, this is automatically not a `host` dependency.

Reply:
In theory we would add a `run_export` section to `cmdstan`, right?

Reply:
Yes I think that would make sense.

Reply:
>In theory we would add a run_export section to cmdstan, right?

I think thats debatable. `run_export` as a kind of ABI/API compatibility feature and also to avoid mistakes down the line. But it doesn't necessarily mean that it's a good idea.